nucleus/annotation.py

```diff
@@ -74,7 +74,7 @@ def __str__(self):
     @classmethod
     def from_json(cls, payload: dict):
         return cls(
-            mask_url=payload[MASK_URL_KEY],
+            mask_url=payload.get(MASK_URL_KEY, None),
```

Comment:
I don't understand, how can you construct a SegmentationAnnotation without a mask_url in the current implementation of `SegmentationAnnotation`? For one, `mask_url`'s type annotation is `str`, so we cannot pass `None` here. And more broadly I don't see how you can have a `SegmentationAnnotation` without a mask. Should be raising an error if MASK_URL_KEY is not in the payload instead?